src/styles/katex-fonts.css

```
@@ -0,0 +1,120 @@
+@font-face {
+	font-family: KaTeX_AMS;
```

Stylelint (17.14.0)

[error] Expected quotes around "KaTeX_AMS" (font-family-name-quotes)

(font-family-name-quotes)